go/vt/vtadmin/api.go

```diff
@@ -796,15 +796,14 @@ func (api *API) VTExplain(ctx context.Context, req *vtadminpb.VTExplainRequest)
 
 	c, ok := api.clusterMap[req.Cluster]
 	if !ok {
-		return nil, errors.ErrUnsupportedCluster
+		return nil, fmt.Errorf("%w: %s", errors.ErrUnsupportedCluster, req.Cluster)
 	}
 
 	tablet, err := c.FindTablet(ctx, func(t *vtadminpb.Tablet) bool {
 		return t.Tablet.Keyspace == req.Keyspace && topo.IsInServingGraph(t.Tablet.Type) && t.Tablet.Type != topodatapb.TabletType_MASTER && t.State == vtadminpb.Tablet_SERVING
 	})
-
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("cannot find serving, non-MASTER tablet in keyspace=%s: %w", req.Keyspace, err)
```

Comment:
How about `non-PRIMARY`?

Reply:
yeah, i was trying to stick to the value of the enum (hence why it's all-caps) but screw it; let's change the english now!